elide-core/src/main/java/com/yahoo/elide/graphql/PersistentResourceFetcher.java

```diff
@@ -531,4 +545,22 @@ private Object replaceObjects(Environment context) {
     private Optional<Sorting> buildSorting(Optional<String> sort) {
         return sort.map(Sorting::parseSortRule);
     }
+
+    private Optional<FilterExpression> buildFilter(String typeName,
+                                                   Optional<String> filter,
+                                                   RequestScope requestScope) {
+        return filter.map(filterStr -> {
+            MultivaluedMap<String, String> queryParams = new MultivaluedHashMap<String, String>() {
+                {
+                    put("filter[" + typeName + "]", Arrays.asList(filterStr));
```

Comment:
Can we refactor the filterDialect so that it just takes a String instead of a map of query params.